src/validate_pyproject/formats.py

```diff
@@ -378,7 +378,31 @@ def int(value: builtins.int) -> bool:
     return -(2**63) <= value < 2**63
 
 
-def SPDX(value: str) -> bool:
-    """Should validate eventually"""
-    # TODO: validate conditional to the presence of (the right version) of packaging
-    return True
+try:
+    try:
+        from packaging import licenses as _licenses
+    except ImportError:  # pragma: no cover
+        # let's try setuptools vendored version
+        from setuptools._vendor.packaging import (  # type: ignore[no-redef]
+            licenses as _licenses,
+        )
+
+    def SPDX(value: str) -> bool:
+        """See :ref:`PyPA's license specifiers <pypa:#license>`
+        (amended in :pep:`639`).
+        """
+        try:
+            _licenses.canonicalize_license_expression(value)
+            return True
+        except _licenses.InvalidLicenseExpression:
+            return False
+
+except ImportError:  # pragma: no cover
+    _logger.warning(
+        "Could not find an installation of `packaging`. License expressions "
+        "might not be validated. "
+        "To enforce validation, please install `packaging`."
```

Comment:
Maybe:

```suggestion
        "Could not find an up-to-date installation of `packaging`. "
        "License expressions might not be validated. "
        "To enforce validation, please install `packaging>=24.2`."
```


We will probably also have to update the extra dependency in `pyproject.toml` to `    "packaging>=24.2"`

Reply:
Isn't that already updated? It's the `[all]` one, right?

Reply:
Perfect.

Sorry for the comment, didn't realise.